### DISCUSSION TITLE
Change the gun mod descrptions

### DISCUSSION THREAD

**Discussion by Unrepentant-Atheist** [cataclysmbn/Cataclysm-BN#3480] [General]
### Is your feature request related to a problem? Please describe.

![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/3938423/097e7768-6660-47c8-8322-e79e5d60058b)

This really rustles me jimmies: `x/x acessories , x/x barrel, ...`

### Describe the solution you'd like

Should be the other way around, like it's usually shown in most games `acessories x/x , barrel x/x, ...`

**Comment by chaosvolt** [comment]
You already brought this up in the discord and it immediately descended into people disagreeing on if that actually looks better or not, so this is 100% a subjective topic that belongs in discussions, not an issue.
![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/11582235/977a969a-224d-45a9-8391-d63837089b59)
